_summaries/wu2021rethinking_2.md

```diff
@@ -1,19 +1,71 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rethinking "Batch" in BatchNorm, Wu, Johnnson; 2021
+paper: wu2021rethinking_2
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: saikm200022
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# **Summary - Rethinking "Batch" in BatchNorm, Wu, Johnnson; 2021**
+
+## What is the core idea?
+
+BatchNorm is a commonly used component of neural networks that has greatly increased performance in image classifcation tasks. However, there are certain inherent aspects of BatchNorm that can lead to sub-optimal performance if not addressed correctly. This paper identifies issues primarily around population statistics computation,  training and test set inconsistency, domain shift, and information leakage. The paper also suggests solutions to combat each of these issues to make the most out of BatchNorm in networks.
+
+![Alt Text](wu2021_1f.PNG)
```

Comment:
I wonder if possible to change the formula to latex, it looks better than the image,

$$y=\frac{x-\mu}{\sqrt{\sigma^2+\epsilon}}$$